packages/main/src/RatingIndicator.ts

```diff
@@ -299,7 +303,7 @@ class RatingIndicator extends UI5Element {
 	}
 
 	get tabIndex() {
```

Comment:
`get tabIndex` conflicts with a built-int TS interface, instead of changing the values, we decided to rename the getters from this kind to `get effectiveTabIndex` as here https://github.com/SAP/ui5-webcomponents/pull/6292/files#diff-03657c030748d0c85f41746bb6ed13f8e1ca2b5ca1436c08dbbf9f0a0f3a453bR443